gcivil-nyu-org/fall2019-cs-gy-6063-team-debug_entities
# PR for #160: Change genres to a many-to-many

### Description

* Changed the genres field of the Concert model to a many-to-many
* Updated the pull_seatgeek_data command to work with this new many-to-many field
* Deleted the genres.txt file because it has been replaced by the Genre model
* Added the Concert model to the admin interface

### Testing Directions
```
python manage.py makemigrations
python manage.py migrate
python manage.py pull_seatgeek_data
python manage.py make_users
```

1.  See that the pull_seatgeek_data command does not give any errors.
1. Look at the Concert model in the admin interface and see that the genres are no longer a string. It now has a multi-select widget because it's now a many-to-many under the hood.
> ![image](https://user-images.githubusercontent.com/13837978/69079680-8a07bb00-0a08-11ea-893b-4ca01ca62042.png)


### Miscellaneous Comments

* These changes broke the genres filter on the events page. That's expected. I'll fix that on #153.

### Checklist
